fs/fs.go

```diff
@@ -413,17 +413,17 @@ func (i *RealFsInfo) GetFsInfoForPath(mountSet map[string]struct{}) ([]Fs, error
 				// if /dev/zfs is not present default to VFS
 				fallthrough
 			case NFS.String():
-				if !utils.FileExists(partition.mountpoint) {
-					klog.V(4).Infof("the file system type is %s, partition mountpoint does not exist: %v", partition.fsType, partition.mountpoint)
-					break
-				}
 				devId := fmt.Sprintf("%d:%d", partition.major, partition.minor)
 				if v, ok := nfsInfo[devId]; ok {
 					fs = v
 					break
 				}
 				var inodes, inodesFree uint64
 				fs.Capacity, fs.Free, fs.Available, inodes, inodesFree, err = getVfsStats(partition.mountpoint)
+				if err != nil {
+					klog.V(4).Infof("the file system type is %s, partition mountpoint does not exist: %v", partition.fsType, partition.mountpoint)
```

Comment:
Can you do something along the lines:
```suggestion
					klog.V(4).Infof("the file system type is %s, partition mountpoint does not exist: %v, error: %s", partition.fsType, partition.mountpoint, err)
```
I don't think we should lose the error causing the message to be logged.

Reply:
done